templates/akeneo/README.patch

```diff
@@ -0,0 +1,75 @@
+ 
+-Scrutinizer | Crowdin
+------------ | -------
+-[![Scrutinizer Quality Score](https://scrutinizer-ci.com/g/akeneo/pim-community-dev/badges/quality-score.png?s=05ef3d5d2bbfae2f9a659060b21711d275f0c1ff)](https://scrutinizer-ci.com/g/akeneo/pim-community-dev/) | [![Crowdin](https://d322cqt584bo4o.cloudfront.net/akeneo/localized.svg)](https://crowdin.com/project/akeneo)
++1. Create a new project through the Platform.sh user interface and select "start new project from a template".  Then select Akeneo as the template. That will create a new project using this repository as a starting point.
+ 
+-Application Technical Information
+----------------------------------
++2. Take an existing project, add the necessary Platform.sh files, and push it to a Platform.sh Git repository.
+ 
+-The following documentation is designed for both clients and partners and provides all technical information required to define required server(s) to run Akeneo PIM application and check that end users workstation is compatible with Akeneo PIM application:
+-https://docs.akeneo.com/master/install_pim/manual/system_requirements/system_requirements.html
++## Using as a reference
+ 
+-Installation instructions
+--------------------------
++You can use this repository as a reference for your own Akeneo projects, and borrow whatever code is needed.  The most important parts are the [`.platform.app.yaml`](/.platform.app.yaml) file and the [`.platform`](/.platform) directory.
+ 
+-### Recommended installation
+ 
+-To install Akeneo PIM for a PIM project or for evaluation, please follow: https://docs.akeneo.com/master/install_pim/manual/installation_ce_archive.html
++Also see:
++ 
++* [`config.yml`](/app/config/config.yml) - At the top of this file in the `imports` section, a new resource is added named `parameters_platform.php`.  That will load a PHP file rather than YAML file to specify Symfony configuration parameters.
++* [`parameters_platform.php`](/app/config/parameters_platform.php) - This file contains Platform.sh-specific code to map environment variables into Symfony parameters. This file will be parsed on every page load. By default it only maps a default database and elasticsearch connection parameters. You can add to it as needed.
++* [`parameters.yml.dist`](/app/config/parameters.yml.dist) - This file is modified so the install process can retrieve the database connection parameters, SwiftMailer can connect to the correct host and the inital data set is set to `minimal`
+ 
+-### Using Composer to create the project
++The template is configured to install the data set automatically during the build hook. That means after the first build and deployment, you can login with user: `admin` / password: `admin`.
+ 
+-Alternatively, you can install Akeneo PIM with Composer, but please make sure that all requirements are fulfilled.
++Please note, that we use a worker for the Akeneo `job-queue-consumer-daemon`. That means you need to run this on a Medium plan or higher.
```

Comment:
"this project uses a worker..."  Avoid using "royal we" if possible.

Reply:
changed it and switched from patch to new file